pkgs/edge-worker/project.json

```
@@ -3,7 +3,32 @@
+        "outputPath": "pkgs/edge-worker/dist",
+        "main": "pkgs/edge-worker/src/index.ts",
+        "tsConfig": "pkgs/edge-worker/tsconfig.lib.json",
+        "platform": "node",
+        "format": ["esm"],
+        "assets": ["pkgs/edge-worker/*.md"],
+        "external": ["@jsr/*", "postgres"],
+        "bundle": true,
+        "sourcemap": true
+      }
+    },
+    "jsr:publish": {
```

Biome (1.9.4)

[error] The key jsr:publish was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)